### PR TITLE
FCL-155 | refactor search boxes

### PR DESCRIPTION
**Pull request by jlhdxw**
## Changes in this PR:

 - Extracting a couple of search related components out
 - Moving shared html into components and removing duplication
 - Sorting our CSS imports and labelling + removing duplication
 - Adding search components to the style guide
 - Adding missing menu items to style guide
 - Because our search components aren't responsive without mediaqueries, I've added a styleguide component wrapper that sets the container to full width with a scroll bar so you can see the component as it's supposed to be seen

## Jira card / Rollbar error (etc)

https://national-archives.atlassian.net/browse/FCL-155

## Screenshots of UI changes:

Results Filters Inputs

image

Search form:

image

Search term:

image

Structured search:

image

### CODE REVIEW

File: ds_judgements_public_ui/sass/main.scss [hunk 1]
```diff
@@ -3,60 +3,79 @@
...
 $govuk-focus-colour: $color-focus-blue-outline;
 @import "../../node_modules/govuk-frontend/dist/govuk/all";
 
-// Components
-@import "includes/animations";
-@import "includes/browse_by_court";
-@import "includes/buttons";
+// Vendor overrides
+@import "includes/govuk_overrides";
+// TODO: Merge this into govuk_overrides
```
Comment (dragon-dxw) [inline ds_judgements_public_ui/sass/main.scss:25]: Is this worth making a ticket for?

Comment (dragon-dxw) [inline ds_judgements_public_ui/sass/main.scss:25]: Is this worth making a ticket for?

**Review by dragon-dxw — APPROVED** [review]
Couple of things I'd like you to double-check you're happy with but lgtm.

File: ds_judgements_public_ui/sass/includes/_results.scss [hunk 2]
```diff
@@ -55,6 +55,7 @@
   }
 
   &__list-layout-container {
+    color: red;
```
[the remark this answers is not in the document]
Comment (jlhdxw) [reply ds_judgements_public_ui/sass/includes/_results.scss:58]: Ahh this is a typo! I was testing something. Thank you for spotting

File: ds_judgements_public_ui/sass/main.scss (hunk 1 above)
Comment (jlhdxw) [reply ds_judgements_public_ui/sass/main.scss]: Have updated in this PR :)

**Pull request merged**